### PR TITLE
feat: add team jwt token

### PR DESCRIPTION
**Pull request by zlwaterfield**
Adding the team jwt token for live event stream auth

### CODE REVIEW

**Review by benjackwhite — CHANGES_REQUESTED** [review]
Can we not call it jwt_token and instead make it more specific like live_events_token 

Other than that 👍

File: posthog/api/team.py [hunk 1]
```diff
@@ -196,6 +201,13 @@ def get_has_group_types(self, team: Team) -> bool:
     def get_groups_on_events_querying_enabled(self, team: Team) -> bool:
         return groups_on_events_querying_enabled()
 
+    def get_jwt_token(self, team: Team) -> Optional[str]:
+        return encode_jwt(
+            {"team_id": team.id},
+            timedelta(days=7),
```
Comment (benjackwhite) [inline posthog/api/team.py:207]: Also probably good to have this as a setting but not the end of the world

**Review by benjackwhite — APPROVED** [review]

File: posthog/api/team.py (hunk 1 above)
Comment (zlwaterfield) [reply posthog/api/team.py:207]: Ah good call, can do.

**Review by fuziontech — APPROVED** [review]
love it
:ship: it

**Pull request closed without merging**